core/controllers/base_test.py

```diff
@@ -394,8 +394,7 @@ def test_keys_in_source_code_match_en(self):
         files_checked = 0
         missing_keys_count = 0
         for directory in dirs_to_search:
-            for root, _, files in os.walk(os.path.join(
-                    os.getcwd(), directory)):
+            for root, _, files in os.walk(os.path.join(os.getcwd(), directory)):
```

Comment:
Discussed with @joshuacano, decided to fix in a future PR since lint checks are passing on Travis and this is holding up a bunch of PRs.